themes/default/content/resources/how-to-build-azure-serverless-application/index.md

```diff
@@ -0,0 +1,256 @@
+});
+
+// Create an API Management instance
+const apimInstance = new apimanagement.ApiManagementService("apim", {
+    resourceGroupName: resourceGroup.name,
+    sku: {
+        name: apimanagement.SkuType.Developer, // Developer tier for testing purpose
+        capacity: 1,
+    },
+});
+
+// Create an API with a function backend
+new apimanagement.Api("exampleApi", {
+    resourceGroupName: resourceGroup.name,
+    serviceName: apimInstance.name,
+    protocols: ["https"],
+    serviceName: functionApp.name,
+    path: "example",
+});
+
+exports.endpoint = pulumi.interpolate`https://${functionApp.defaultHostName}/api`;
+
+
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.AzureNative.Resources;
+using Pulumi.AzureNative.Storage;
+usingulumi.AzureNative.Storage.Inputs;
+using Pulumi.AzureNative.Web;
+using Pulumi.AzureNative.Web.Inputs;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // Create an Azure Resource Group
+        var resourceGroup = new ResourceGroup("resourceGroup");
+
+        // Create an Azure Storage Account
+        var storageAccount = new StorageAccount("storageAccount", new StorageAccountArgs
+        {
+            ResourceGroupName = resourceGroup.Name,
+            Sku = new SkuArgs
+            {
+                Name = SkuName.Standard_LRS
+            },
+            Kind = Kind.StorageV2
+        });
+
+        // Create a blob container in the storage account
+        var blobContainer = new BlobContainer("blobContainer", new BlobContainerArgs
+        {
+            AccountName = storageAccount.Name,
+            ResourceGroupName = resourceGroup.Name,
+            PublicAccess = PublicAccess.None
+        });
+
+        // Create a Consumption Plan for Azure Functions
+        var consumptionPlan = new AppServicePlan("consumptionPlan", new AppServicePlanArgs
+        {
+            ResourceGroupName = resourceGroup.Name,
+            Kind = "FunctionApp",
+            Sku = new SkuDescriptionArgs
+            {
+                Tier = "Dynamic",
+                Name = "Y1",
+            },
+        });
+
+        // Create an Azure Function App
+        var functionApp = new WebApp("functionApp", new WebAppArgs
+        {
+            ResourceGroupName = resourceGroup.Name,
+            ServerFarmId = consumptionPlan.Id,
+            Kind = "functionapp",
+            SiteConfig = new SiteConfigArgs
+            {
+                AppSettings = new []
+                {
+                    new NameValuePairArgs { Name = "AzureWebJobsStorage", Value = storageAccount.PrimaryConnectionString },
+                    new NameValuePairArgs { Name = "WEBSITE_RUN_FROM_PACKAGE", Value = "1" },
+                }
+            }
+        });
+    }
+}
+
+```
+{{% /choosable %}}
+{{% choosable language yaml %}}
+
+```yaml
+
+apiVersion: pulumi.com/v1alpha1
```

Comment:
just for my own edification ... what exactly is "v1alpha1" referring to here?

Reply:
this is an example of a PulumiAI hallucination; our kubernetes docs reference v1alpha : i.e. https://www.pulumi.com/registry/packages/kubernetes/api-docs/settings/v1alpha1/